airbyte-integrations/connectors/source-oracle-v2/src/cdk/test/resources/command/expected-schema.json

```diff
@@ -0,0 +1,214 @@
+{
```

Comment:
I'v'e always been kind of annoyed by these expected-schema tests.  
* In the old days, we were just testing that file A looked like file B - that's not helpful
* In this new world where specs are generated by code... do we care here?  A test in the CDK makes sense, but I think all we need to test for in the connector is that SPEC worked, not the content.

Reply:
The purpose of this test is to guard against any unexpected changes, and in the case of expected changes, to visualize how they're reflected from the config POJO to its json schema. A PR which makes a schema change therefore will also include a few line changes in this file in this diff, which I find desirable.

Reply:
yeah, I'll second Marius here. I think it becomes easy to modify a class without realizing that it'll. modify the spec.json. Having a gold file test makes sense when one of the files is generated, or modified. Makes a lot less sense when both files are written as-is